docs/flow/modin/experimental/range_partitioning_groupby.rst

```diff
@@ -78,3 +78,9 @@ Range-partitioning Merge
 
 It is recommended to use this implementation if the right dataframe in merge is as big as
 the left dataframe. In this case, range-partitioning implementation works faster and consumes less RAM.
+
+'.unique()' and '.drop_duplicates()'
```

Comment:
Should we refactor this doc page for 0.29.0?

Reply:
Yes, and there is an issue for that #6987